frontend-react/e2e/spec/all/public/about/case-studies.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { AboutCaseStudiesPage } from "../../../../pages/public/about/case-studies";
+import { test as baseTest, expect } from "../../../../test";
+
+export interface Fixtures {
+    aboutCaseStudiesPage: AboutCaseStudiesPage;
+}
+
+const test = baseTest.extend<Fixtures>({
+    aboutCaseStudiesPage: async (
+        {
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        },
+        use,
+    ) => {
+        const page = new AboutCaseStudiesPage({
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        });
+        await page.goto();
+        await use(page);
+    },
+});
+
+test.describe(
+    "About / Case Studies page",
+    {
+        tag: "@smoke",
+    },
+    () => {
+        test("has correct title", async ({ aboutCaseStudiesPage }) => {
+            await expect(aboutCaseStudiesPage.page).toHaveTitle(aboutCaseStudiesPage.title);
+            await expect(aboutCaseStudiesPage.heading).toBeVisible();
+        });
+
+        test("has side nav", async ({ aboutCaseStudiesPage }) => {
+            await expect(aboutCaseStudiesPage.page.getByRole("navigation", { name: "side-navigation " })).toBeVisible();
```

Comment:
Please check here for an example on how to use the new `.testSidenav()` method from `BasePage`: https://github.com/CDCgov/prime-reportstream/blob/2f0682e8e5a75ba4cfa7b59e0246560d54b989ad/frontend-react/e2e/spec/all/public/about/roadmap.spec.ts#L64

Reply:
This applies to frontend-react/e2e/spec/all/public/about/release-notes.spec.ts as well.